### DISCUSSION TITLE
Allow host to enable 'perks' in Sandbox / Mission mode too.

### DISCUSSION THREAD

**Discussion by Ek-Videogames** [FakeFishGames/Barotrauma#15294] [Ideas and suggestions]
The perk system is great for allowing loadouts and customization to PVP rounds.

The same thing would actually be a great option for Sandbox & Missions too, allowing hardpoints to be swapped outside of campaign, and new items to be purchased - without having to use the cumbersome 'spawn additional cargo' system.

**Comment by AziKarnage** [comment]
> The perk system is great for allowing loadouts and customization to PVP rounds.
> 
> The same thing would actually be a great option for Sandbox & Missions too, allowing hardpoints to be swapped outside of campaign, and new items to be purchased - without having to use the cumbersome 'spawn additional cargo' system.

Are you the guy who made the EK mod?

**Comment by Regalis11** [comment]
The perks have been usable in sandbox and mission mode for some time, looks like we've just missed this ticket :barodev:
